app/controllers/bulkrax/exporters_controller_decorator.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Override Bulkrax main (at v5.3.0) to limit access to only user's own exporters
+module Bulkrax
+  module ExportersControllerDecorator
+    def index
+      # NOTE: We're paginating this in the browser.
+      @exporters = Exporter.order(created_at: :desc)
+      @exporters = @exporters.where(user: current_user) unless current_ability.admin?
+      @exporters = @exporters.all
```

Comment:
We can condense these queries so that we're only hitting the database once (the same idea can be applied to the importers controller): 

```suggestion
      @exporters = if current_ability.admin?
        Exporter.order(created_at: :desc)
      else
        Exporter.where(user: current_user).order(created_at: :desc)
      end
```

Reply:
@bkiahstroud in the original lines 8 and 9 do not hit the database.  They are building the ActiveRecord Relation clause which will be used to fetch from the database.  On line 10 is when we first hit the Exporter database.

I suspect we don't need to include the `.all` and could let the views resolve that.  However, the `.all` call was part of the prior implementation so we're leaving it here.